packages/validator/src/metrics.ts

```diff
@@ -169,6 +169,12 @@ export function getMetrics(register: MetricsRegister, gitData: LodestarGitData)
       help: "Total published aggregates",
     }),
 
+    numParticipantsInAggregate: register.histogram({
+      name: "vc_attestation_service_participants_in_aggregate_total",
+      help: "Number of attestations in the published AggregatedAttestation",
+      buckets: [0, 50, 200, 500],
```

Comment:
Would be good to refine this buckets after watching metrics for a bit

Reply:
what I see on goerli is not great, it's 60-70 maximum
right now a sync committee on mainnet has 219 members, on goerli it's 189 and they'll gradually grow

Reply:
You mean beacon attestation committee?